### ISSUE TITLE
Documentation of TF Deep Explainer

### ISSUE THREAD

**Issue by farh33n** [shap/shap#943]
I would really appreciate if documentation of TF Deep Explainer could be made more detailed. I am having a difficult time understanding the linearity and non linearity handlers and also I couldn't figure out where are the shap values actually computed. How is this tf.gradient different from the regular gradient? I would truly appreciate if some one could help me with it
P.S. I have read both Shap and Deep Lift papers.

**Comment by slundberg** [comment]
We are working on updating the DeepExplainer for TF 2.0, I'll leave this as a todo for after that gets completed.

**Comment by farh33n** [comment]
> We are working on updating the DeepExplainer for TF 2.0, I'll leave this as a todo for after that gets completed.

When do you expect it to be completed?

**Comment by slundberg** [comment]
I would be surprised if it took more than a month. It is a bit hard to predict with Christmas and New years coming up though.

**Comment by philippatt** [comment]
Hi, is the DeepExplainer for TF2.0 got updated? I am using tf.keras, and looks like the SHAP package only works for keras, but not the model trained uding tf.keras. Please advise. Thanks you!
